src/main/kotlin/se/uulm/snowballr/backend/validation/ValidationHelper.kt

```diff
@@ -141,3 +145,18 @@ fun Raise<ValidationIssue>.ensureLastNameValidity(lastName: String) {
     ensureFieldNonBlank("last_name", lastName)
     ensureFieldLength("last_name", lastName, LAST_NAME_MAX_LENGTH)
 }
+
+/**
+ * Ensures that the provided field mask contains only valid fields for the given object type and
+ * is non-blank.
+ *
+ * @param fieldMask The FieldMask to validate.
+ * @param descriptor The object descriptor to validate against.
+ * @throws ValidationIssue if the field mask contains invalid or blank fields.
+ */
```

Comment:
We don't throw anything here.
```suggestion
 * @param descriptor The object descriptor to validate against.
 */
```

Reply:
You are right. Initially, I thought that this raise type was just a fancier exception, but now I understand better what it is.